coffe/hardblock_functions.py

```diff
@@ -344,7 +524,8 @@ def run_sim():
 
 def write_pt_power_script(flow_settings,mode_enabled,clock_period,x):
   """"
-  writes the tcl script for timing analysis using Synopsys Design Compiler, tested under 2017 version
+  writes the tcl script for power analysis using Synopsys Design Compiler, tested under 2017 version
+  Update: changed the [all_nets] argument used in set_switching_activity to use -baseClk and -inputs args instead
```

Comment:
If you know how this does power analysis it is good to comment it. (If not, don't worry about it). 
From reading the code below it looks like it will read in a switching activity file if the "generate_activity_file" flag was set (at the command line I assume?) or uses the user-specified switching activity otherwise.